module-gui/gui/widgets/text/core/lines/MultiTextLine.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "MultiTextLine.hpp"
+
+namespace gui
+{
+    /// Note - line breaking could be done here with different TextLines to return
+    /// or via different block types (i.e. numeric block tyle could be not "breakable"
+    MultiTextLine::MultiTextLine(BlockCursor &localCursor, unsigned int maxWidth) : TextLine(localCursor, maxWidth)
+    {
+        do {
+            if (!localCursor) { // cursor is faulty
```

Comment:
if it's faulty consider `assert` at least?

Reply:
Hmm that is rly old copied code :). Will check if assert wont break code (it can imo).

Reply:
yep -> assert is breaking drawing. That is valid exit condition and can't be asserted.